python_ta/__init__.py

```diff
@@ -244,11 +244,15 @@ def _override_config(linter: PyLinter, config_location: AnyStr) -> None:
         print(ex, file=sys.stderr)
         sys.exit(32)
 
+    unrecognized_options_message = None
     # Override the config options by parsing the provided file.
     try:
         linter._parse_configuration_file(config_args)
     except _UnrecognizedOptionError as exc:
-        print(f"Unrecognized options: {', '.join(exc.options)}", file=sys.stderr)
+        unrecognized_options_message = ", ".join(exc.options)
```

Comment:
It's a bit strange to just set this variable here. You can instead call `linter.add_message` directly in this `except` block.